samples/drivers/watchdog/sample.yaml

```diff
@@ -13,9 +13,10 @@ tests:
             - "Waiting for reset..."
             - "Watchdog sample application"
     depends_on: watchdog
-    filter: not dt_compat_enabled("st,stm32-window-watchdog")
+    filter: not CONFIG_SOC_FAMILY_STM32
```

Comment:
Would this change not disable testing of this sample for the IWDG? Was that the intention?

Reply:
When running on stm32 platforms, the  	`aliases {	watchdog0 = };` must be defined to be either` &iwdg`;   or` &wwdg; ` and  creating two special configurations through overlay files.
Other test config  _sample.drivers.watchdog_   is then no more relevant  for stm32devices.

Reply:
BTW if you have any idea (for the sample.yaml I guess)  to avoid _non CONFIG_SOC_FAMILY_STM32_ boards to build (and consequently fail) sample.drivers.watchdog.stm32wwdg or sample.drivers.watchdog.stm32iwdg, that could help to pass the CI Checks " Run tests with twister / twister-build (1) (pull_request_target) Failing after 11m — twister-build (1)"